molgenis-swagger/src/main/resources/templates/view-swagger.ftl

```diff
@@ -0,0 +1,525 @@
+          required: true
+        - name: _method
+          type: string
+          in: query
+          enum:
+            - POST
+            - GET
+          description: Tunnel request through defined method over default API operation
+      responses:
+        200:
+          description: OK
+  /v2/copy/{entity_name}:
+    post:
+      tags:
+        - V2
+      summary: Creates a copy of an entity.
+      description: The copy will be created in the same package and backend as the original entity, so both must be writable.
+      parameters:
+        - name: entity_name
+          in: path
+          type: string
+          description: Name of the entity
+          required: true
+          enum:
+<#list entityTypes as entityType>
+            - ${entityType}
+</#list>
+        - name: body
+          schema:
+            $ref: "#/definitions/CopyEntityRequest"
+          in: body
+          required: true
+      responses:
+        400:
+          description: For an unknown entity.
+          schema:
+            $ref: "#/definitions/ErrorMessageResponse"
+        401:
+          description: If the user lacks the proper permissions.
+          schema:
+            $ref: "#/definitions/ErrorMessageResponse"
+        500:
+          description: If a runtime exception occurs.
+          schema:
+            $ref: "#/definitions/ErrorMessageResponse"
+definitions:
+  CopyEntityRequest:
+    type: object
+    properties:
+      newEntityName:
+        type: string
+    required:
+      - newEntityName
+  LoginRequest:
+    type: object
+    properties:
+      username:
+        type: string
+      password:
+        type: string
+  LoginResponse:
+    type: object
+    properties:
+      token:
+        type: string
+      username:
+        type: string
+  EntityCollectionRequestV2:
+    type: object
+    properties:
+      q:
+        type: string
+      aggs:
+        type: string
+      sort:
+        type: string
+      attrs:
+        type: string
+      start:
+        type: integer
+        format: int32
+        default: 0
+        minimum: 0
+        description: Offset in resource collection
+      num:
+        type: integer
+        format: int32
+        default: 0
+        minimum: 0
+        maximum: 10000
+        description: Number of resources to retrieve starting at start
+  EntityCollectionResponseV2:
+    type: object
+    properties:
+      href:
+        type: string
+      meta:
+        $ref: "#/definitions/EntityTypeResponseV2"
+      start:
+        type: integer
+        format: int32
+      num:
+        type: integer
+        format: int32
+      total:
+        type: integer
+        format: int64
+      prevHref:
+        type: string
+      nextHref:
+        type: string
+      items:
+        type: array
+        items:
+          type: object
+    required:
+      - href
+      - meta
+      - start
+      - num
+      - total
+      - items
+  EntityTypeResponseV2:
```

Comment:
Add notes to AttributeResponseV2 and EntityTypeResponseV2 stating that adding properties to this classes (e.g. when we introduce requiredExpression) requires updating this file.